### PR TITLE
Made scenario1_5 independent from scenario1

### LINKED ISSUE #180: AWX fix for scenario1

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
Closes #180

### CODE REVIEW

File: scenarios/modules/loadbalancer/loadbalancer.tf [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+output "scn1_5_lb_fip" {
+  value = opentelekomcloud_compute_floatingip_v2.lb_public_ip.address
+}
+
+# Create listener
+resource "opentelekomcloud_lb_listener_v2" "listener" {
+  name            = "${var.workspace_prefix}${var.scenario}_listener_http"
+  protocol        = var.protocol
+  protocol_port   = 80
```
Comment (outcatcher) [inline scenarios/modules/loadbalancer/loadbalancer.tf]: Maybe move `protocol_port` to variables (or locals)?

File: scenarios/modules/loadbalancer/loadbalancer.tf [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  depends_on = [
+    opentelekomcloud_lb_pool_v2.pool
+  ]
+}
+
+# Create health monitor for check services instances status
+resource "opentelekomcloud_lb_monitor_v2" "monitor" {
+  name        = "${var.workspace_prefix}${var.scenario}_monitor_http"
+  pool_id     = opentelekomcloud_lb_pool_v2.pool.id
+  type        = "TCP"
```
Comment (outcatcher) [inline scenarios/modules/loadbalancer/loadbalancer.tf]: It seems, `protocol` variable meant to be used here

File: scenarios/scenario1_5/resources/vm.tf [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+# Create instance
+resource "opentelekomcloud_compute_instance_v2" "http" {
+  count       = length(opentelekomcloud_networking_port_v2.http)
+  name        = "${local.workspace_prefix}${var.scenario}_basic_${count.index}"
+  flavor_name = var.ecs_flavor
+  key_pair    = var.key_pair_name
+  user_data   = file("${path.module}/first_boot.sh")
+
+  availability_zone = "eu-de-03"
```
Comment (outcatcher) [inline scenarios/scenario1_5/resources/vm.tf]: Move AZ to variables?

File: scenarios/modules/loadbalancer/variables.tf [hunk 4]
```diff
@@ -0,0 +1,13 @@
+variable "scenario" {}
+variable "subnet_id" {}
+variable "loadbalancer_local_ip" {}
```
Comment (outcatcher) [inline scenarios/modules/loadbalancer/variables.tf]: Maybe pass only subnet 3 octets to loadbalancer module? And return local IP if needed?

File: scenarios/scenario1_5/resources/vm.tf [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  security_group_ids = [
+    var.bastion_sec_group_id
+  ]
+  fixed_ip {
+    subnet_id  = var.subnet_id
+    ip_address = "${var.net_address}.${count.index + 10}"
+  }
+}
+
+output "ports" {
```
Comment (outcatcher) [inline scenarios/scenario1_5/resources/vm.tf]: Are `ports` used outside of module?

File: scenarios/modules/loadbalancer/loadbalancer.tf [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+  listener_id = opentelekomcloud_lb_listener_v2.listener.id
+  depends_on = [
+    opentelekomcloud_lb_listener_v2.listener
+  ]
+}
+
+# Add multip instances to pool
+resource "opentelekomcloud_lb_member_v2" "members" {
+  count         = var.nodes_count
+  address       = "${var.net_address}.${count.index + 10}"
```
Comment (outcatcher) [inline scenarios/modules/loadbalancer/loadbalancer.tf]: Can this info be retrieved from instances?

File: scenarios/modules/loadbalancer/loadbalancer.tf [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+resource "opentelekomcloud_compute_floatingip_v2" "lb_public_ip" {}
+
+# Assign FIP to Loadbalancer
+resource opentelekomcloud_networking_floatingip_associate_v2 "floatingip_associate_lb" {
+  floating_ip = opentelekomcloud_compute_floatingip_v2.lb_public_ip.address
+  port_id     = opentelekomcloud_lb_loadbalancer_v2.loadbalancer.vip_port_id
+}
+
+output "scn1_5_lb_fip" {
```
Comment (lego963) [inline scenarios/modules/loadbalancer/loadbalancer.tf]: name links to concrete scenario

**Pull request merged**